tools/editor/project_manager.cpp

```diff
@@ -1126,10 +1126,10 @@ ProjectManager::ProjectManager() {
 
 
 	if (StreamPeerSSL::is_available()) {
-
 		asset_library = memnew( EditorAssetLibrary(true) );
 		asset_library->set_name("Templates");
 		tabs->add_child(asset_library);
+        asset_library->connect("install_asset",this,"_install_project");
```

Comment:
Could you fix the indentation to use tabs instead of spaces? (And when doing so, please use `git commit --amend` or `git rebase -i HEAD~2` after committing to squash the two commits together).